doc/riscv-bfloat16-zfbfinxmin.adoc

```diff
@@ -0,0 +1,29 @@
+[[zfbfinxmin,Zfbfinxmin]]
+=== `Zfbfinxmin` - Scalar BF16 Converts In Integer Registers
+
+This extension provides the minimal set of instructions needed to enable scalar support
+of the BF16 format on platforms using the `Zfinx` family of extensions (where
+floating point instructions operate on `x` registers rather than `f`
+registers). It enables BF16 as an interchange format as it provides conversion
+between BF16 values and FP32 values. It requires the `Zfinx` extension.
+
+As `FLH`, `FSH`, `FMX.X.H`, and `FMX.H.X` are unnecessary in the absence of
+`f` registers, it defines only two instructions: `FCVT.BF16.S` and
+`FCVT.S.BF16`. These instructions have the same semantics as they do within
+the `Zfbfmin` extension, except that whenever such an instruction would have
+accessed an `f` register, it instead accesses the `x` register with the same
+number.
+
+As with the other floating-point in integer register extensions, floating
+point operands of width `w < XLEN` bits occupy bits `w-1:0` of an `x`
+register.  Floating-point operations on `w`-bit operands ignore operand bits
+`XLEN-1:w`. Floating point operations that produce `w < XLEN`-bit results fill
+`XLEN-1:w` with copies of bit `w-1` (the sign bit).
+[%autowidth]
+[%header,cols="2,4"]
+|===
+|Mnemonic
+|Instruction
+|FCVT.BF16.S    | <<insns-fcvt.bf16.s>>
+|FCVT.S.BF16    | <<insns-fcvt.s.bf16>>
```

Comment:
Would FCVT.S.BF16 be equivalent to SLLI rd, rs1, 16?

Reply:
How does NaN boxing works on Zdinx/Zfinx, does we need to inject 1s into the 32 MSBs in that case ? (which would not be handled by the `SLLI` if the input was not a proper single precision NaN-boxed float to begin with)

Reply:
There are no FP load instructions with Zfinx so there's no guarantee that an input operand is properly nan-boxed for any FP instruction. So the hardware can't check that on inputs. FP instructions do still produce nan-boxed outputs.

So FCVT.BF16.S would not be exactly equivalent to SLLI for the upper bits. But since nan-boxing is already weakened for Zfinx is that important?